pkg/skaffold/schema/latest/config.go

```diff
@@ -408,6 +408,10 @@ type KubectlFlags struct {
 
 	// Delete are additional flags passed on deletions (`kubectl delete`).
 	Delete []string `yaml:"delete,omitempty"`
+
+	// DisableValidation passes the `--validate=false` flag to supported
+	// `kubectl` commands when enabled.
+	DisableValidation bool `yaml:"validate,omitempty"`
```

Comment:
Thank you for making all the changes!
One more problem and we should be good to go:
`DisableValidation` can't be called `validate` in the yaml. The first one is a negation. `disableValidation` seems an ok name

Reply:
Thanks for your review and patience as I try to squeeze this in my mornings :-)